chapters/install/index.rst

```diff
@@ -1,10 +1,46 @@
-Installing Jina Core
+Installation
 ======================================
 
+Install using `pip`
```

Comment:
I think it should be clear that this is for Linux & Mac. To give context about why windows install is sepreate.